content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -3,399 +3,1034 @@ title: Using Windows Server Containers in Kubernetes
+Key Kubernetes elements work the same way in Windows as they do in Linux. In this section, we will talk about some of the key workload enablers and how they map to Windows.
+
+* [Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/)
+
+    A Pod is the basic building block of Kubernetes–the smallest and simplest unit in the Kubernetes object model that you create or deploy. The following Pod capabilities, properties and events are supported with Windows containers:
+
+  * Single or multiple containers per Pod with process isolation and volume sharing
+  * Pod status fields 
+  * Readiness and Liveness probes
+  * postStart & preStop container lifecycle events
+  * ConfigMap, Secrets: as environment variables or volumes
+  * EmptyDir
+  * Named pipe host mounts
+  * Resource limits
+* [Controllers](https://kubernetes.io/docs/concepts/workloads/controllers/)
+
+    Kubernetes controllers handle the desired state of Pods. The following workload controllers are supported with Windows containers:
+
+  * ReplicaSet
+  * ReplicationController
+  * Deployments
+  * StatefulSets
+  * DaemonSet
+  * Job
+  * CronJob
+* [Services](https://kubernetes.io/docs/concepts/services-networking/service/)
+
+    A Kubernetes Service is an abstraction which defines a logical set of Pods and a policy by which to access them - sometimes called a micro-service. You can use services for cross-operating system connectivity. In Windows, services can utilize the following types, properties and capabilities:
+
+  * Service Environment variables
+  * NodePort
+  * ClusterIP
+  * LoadBalancer
+  * ExternalName
+  * Headless services
+
+Pods, Controllers and Services are critical elements to managing Windows workloads on Kubernetes. However, on their own they are not enough to enable the proper lifecycle management of Windows workloads in a dynamic cloud native environment. We added support for the following features:
+
+* Pod and container metrics
+* Horizontal Pod Autoscaler support
+* kubectl Exec
+* Resource Quotas
+* Scheduler preemption
+
+#### Container Runtime
+
+Docker EE-basic 18.09 is required on Windows Server 2019 / 1809 nodes for Kubernetes. This works with the dockershim code included in the kubelet. Additional runtimes such as CRI-ContainerD may be supported in later Kubernetes versions.
+
+#### Storage
+
+Kubernetes Volumes enable complex applications with data persistence and Pod volume sharing requirements to be deployed on Kubernetes. Kubernetes on Windows supports the following types of [volumes](https://kubernetes.io/docs/concepts/storage/volumes/):
+
+* FlexVolume out-of-tree plugin with [SMB and iSCSI](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows)support
+* [azureDisk](https://kubernetes.io/docs/concepts/storage/volumes/#azuredisk)
+* [azureFile](https://kubernetes.io/docs/concepts/storage/volumes/#azurefile)
```

Comment:
What about the other in-tree plugins like GCE PD/EBS/etc? GCE-PD support for Windows was introduced for 1.14: https://github.com/kubernetes/kubernetes/pull/74612

Reply:
Should there also be a note indicating support for CSI node plugins for Windows will be added in a later release?

Reply:
Found the note for CSI under Restrictions below.

Reply:
yep, we had added the CSI later on :)
i added GcePD as well. we missed that one.
i am not sure about ebs in-tree...not sure if anyone has tried that